apps/health_campaign_field_worker_app/lib/widgets/beneficiary/resource_beneficiary_card.dart

```diff
@@ -70,7 +79,7 @@ class _ResourceBeneficiaryCardState
                     formControlName: 'resourceDelivered.${widget.cardIndex}',
                     valueMapper: (value) {
                       return localizations.translate(
-                        value.sku ?? value.id,
+                        convertForLocalization(value.sku) ?? value.id,
```

Comment:
@amitverma-egov @yash-egov why do we need to convert what is the change we are trying here?? we can change the sku itself in the backend

Reply:
sku contains SPAQ-1 ( 3 a 11 messes) but we need to show only SPAQ-1